### PR TITLE
Connected the Delete Service Function and Built a popup controller class

### PR DESCRIPTION
**Pull request by rkallby**
Built SuccessForm
Built PopupControl class
Moved and renamed the old error message form 
all were moved to a "Popup" folder in gui
Connected Delete Functionality and cleaned up the logic to check the fields before submission.
Swapped all old Error popup code with new call to the popup class controller.

### CODE REVIEW

File: ChocAnonGUI/GUI/Operator/EditServiceForm.cs [hunk 1]
```diff
@@ -55,7 +58,7 @@ private void serviceeditButton_Click(object sender, EventArgs e)
 
             if (servicenameTextbox.Text == "" || servicefeeTextbox.Text == "")
             {
-                MissingEntryForm launchMissingEntry = new MissingEntryForm();
```
Comment (jhinkle97) [inline ChocAnonGUI/GUI/Operator/EditServiceForm.cs:58]: Is there another MissingEntryForm on line 66 that needs replaced?

File: ChocAnonGUI/GUI/Operator/EditServiceForm.cs [hunk 2]
```diff
@@ -33,6 +30,12 @@ private void searchService(object sender, EventArgs e)
           
             ServiceDirectoryController serviceDirectoryController = new ServiceDirectoryController();
             service = serviceDirectoryController.GetService(serviceCode);
+            if (service.Name == "" || service.Fee == "")
+            {
+                disableFields();
+                errorForm.message = "Not a valid Service ID";
```
Comment (jhinkle97) [inline ChocAnonGUI/GUI/Operator/EditServiceForm.cs]: Looks like value is set but never used?

File: ChocAnonGUI/GUI/Operator/EditServiceForm.cs [hunk 3]
```diff
@@ -16,15 +16,12 @@ namespace ChocAnonGUI.GUI.Operator
     public partial class EditServiceForm : Form
     {
         private ServiceDirectoryModel service;
-
+        private ErrorForm errorForm;
```
Comment (jhinkle97) [inline ChocAnonGUI/GUI/Operator/EditServiceForm.cs]: Why is this a global variable?

File: ChocAnonGUI/GUI/Operator/EditServiceForm.cs (hunk 1 above)
Comment (rkallby) [reply ChocAnonGUI/GUI/Operator/EditServiceForm.cs:58]: Shouldn't be

Comment (rkallby) [reply ChocAnonGUI/GUI/Operator/EditServiceForm.cs:58]: That was a removed line..

File: ChocAnonGUI/GUI/Operator/EditServiceForm.cs (hunk 2 above)
Comment (rkallby) [reply ChocAnonGUI/GUI/Operator/EditServiceForm.cs]: It gets used if a person searches for an Service ID, hits edit, removes all the characters in the price and service name fields and decides to switch back to deleting the service

**Review by jhinkle97 — COMMENTED** [review]
All of the changes make sense now with that second commit. Can we change the edit form to just have an edit and delete button side by side instead of a radio button? I think that makes more sense from a UI perspective. We can discuss in our meeting if you don't agree

**Pull request merged**